edb/load_ext/main.py

```diff
@@ -0,0 +1,156 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2016-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Command to load an extension into an edgedb installation.
+
+It is a command distributed with the server, but it is designed so
+that it has no dependencies and does not import any server code
+if it is *only* installing the postgres part of an extension with
+a specified pg_config, so it *can* be pulled out and used standalone.
+(It requires Python 3.11 for tomllib.)
+"""
+
+from __future__ import annotations
+
+
+import argparse
+import os
+import pathlib
+import shutil
+import subprocess
+import sys
+import tomllib
+import zipfile
+
+# Directories that we map to config values in pg_config.
+CONFIG_PATHS = {
+    'share': 'sharedir',
+    'lib': 'libdir',
+    'include': 'includedir',
+}
+
+
+def install_pg_extension(
+    pkg: pathlib.Path,
+    pg_config: dict[str, str],
+) -> None:
+    with zipfile.ZipFile(pkg) as z:
+        with z.open('MANIFEST.toml') as m:
```

Comment:
I'd consider renaming to `GEL_EXTENSION.toml`. cc @elprans

Reply:
I don't see a need to be this specific, it's a generated file in a tarball.

Reply:
Currently it's a hand-written file, though it could also be generated by the build system